ludwig/datasets/mixins/kaggle.py

```diff
@@ -0,0 +1,38 @@
+import os
+from pathlib import Path
+from kaggle.api.kaggle_api_extended import KaggleApi
+
+DEFAULT_CACHE_LOCATION = str(Path.home().joinpath('.ludwig_cache'))
+api = KaggleApi()
+api.authenticate()
+
+class KaggleMixin:
```

Comment:
Let's call this `KaggleDownloadMixin` for consistency with the others.

Reply:
Changed